index.html

```diff
@@ -1383,6 +1383,16 @@ <h2>
 property if they are not encoded in JWK format.
             </td>
           </tr>
+          <tr>
+            <td>
+secp256r1
+            </td>
+            <td>
+Secp256r1 public key values MUST be encoded as the raw 32-byte public key value
```

Comment:
```
Secp256r1 public key values MUST be encoded as the raw 32-byte public key value
encoded in Base58 Bitcoin format using the <code>publicKeyBase58</code> property
if they are not encoded in JWK format.
```
->
```
Secp256r1 public key values MUST be encoded in JSON Web Key (JWK) format using
the <code>publicKeyJwk</code> property or with the raw 32-byte public key
value encoded in Base58 Bitcoin format using the <code>publicKeyBase58</code>
property.
```

Reply:
While I agree with your wording, the consensus position of the group on @selfissued's proposal was to not word things in the way that you worded them. What you are suggesting is more or less what we had before, and the group straw polled to change it so that we don't have normative statements about JWK's usage in the table.

Reply:
I do not recall such a straw poll (which [you refer to nearby as a resolution](https://github.com/w3c/did-core/pull/100#discussion_r351089798), where you cite the special Monday call, which I was not able to join). I do not think I would have agreed with it, given the result, and I do not recall it being mentioned in the regular Tuesday call. Can you provide a specific pointer to the relevant section of minutes?

In any case -- I'm suggesting the text be changed because the current wording is clumsy at best, and confusing at worst.  Either the table should *only* mention the alternative encodings (leaving all mention of JWK in the preceding text), *or* it should be fully explicit (and mention both/all encoding options for each public key in the row), as in the table text I've suggested.